### DISCUSSION TITLE
Please write the lanauges that you want to learn!

### DISCUSSION THREAD

**Discussion by Prokuma** [NAIST-SE/AttractingContributors#880] [2023 Attraction Contributions]
Please write the lanauges that you want to learn (Please write natural language only)

https://github.com/Prokuma/AttractingContributors-Language

**Comment by zihang-wang-naist** [comment]
I added this. 
Could you help me add the travel place you want to visit to the following link
https://github.com/zihang-wang-naist/AttractingContribu-Travel
Thank you

**Comment by chariot-524** [comment]
あなたのリストに中国語を追加するプルリクエストをしました。
私のところにもプルリクエストしてもらえないでしょうか？あなたにとっての最高のゲームのタイトルを追加してほしいです。
https://github.com/chariot-524/AttractingContributors-Game

I made a pull request to add Chinese to your list.
Could you add your best game to the list at the link below?
https://github.com/chariot-524/AttractingContributors-Game
